version.sbt

```diff
@@ -1 +0,0 @@
-version in ThisBuild := "0.1.0-SNAPSHOT"
```

Comment:
is this not needed anymore?

Reply:
Yep :)

With `sbt-ci-release` releases are handled via [tags ](https://github.com/olafurpg/sbt-ci-release#git) (either creating them locally and pushing to origin or creating them through Github releases).

Under the hood `sbt-ci-release` is using [`sbt-dynver`](https://github.com/dwijnand/sbt-dynver) to automatically handle version based on information extracted from Git, and as said in its `README`:

> Then make sure to **NOT** set the version setting, otherwise you will override `sbt-dynver`.